provisioning/e2e/_provisioning_e2e.js

```diff
@@ -130,99 +133,81 @@ var X509Group = function() {
   this.transports = [ Http ];
   var id = uuid.v4();
 
-  this._deviceId = 'deleteme_provisioning_node_e2e_' + id;
   this._registrationId = 'reg-' + id;
-  this._groupId = 'group-' + id;
+  this._groupId = 'testgroup';
 
   this.initialize = function(callback) {
-    debug('creating CA cert');
-    var caConfig = [
-      '[req]',
-      'req_extensions = v3_req',
-      'distinguished_name = req_distinguished_name',
-      '[req_distinguished_name]',
-      'commonName = IoTHub Test CA',
-      'commonName_max = 64',
-      '[v3_req]',
-      'basicConstraints = critical,CA:TRUE'
-    ].join('\n');
-    pem.createCertificate({ config: caConfig, commonName: 'IoTHub Test CA' }, function (err, caCert) {
+    debug('creating device cert');
+    self._factoryCert = {
```

Comment:
>_factoryCert [](start = 9, length = 12)

is that using the CA root cert for something else than testing the CA cert functionality?
I'd rather have separate certs for various purposes. easier to rotate/maintain?

Reply:
also, Doesn't that add to the pile of reasons why an external contributor wouldn't be able to run our tests?

---
In reply to: [154234983](https://github.com/Azure/azure-iot-sdk-node/pull/166#discussion_r154234983) [](ancestors = 154234983)

Reply:
good idea to use a different variable.  I'll switch that around when I actually turn this on (very soon).  It does add overhead to external contributors.  The certGen.sh script that makes this is in the c repo, so it shouldn't be too hard to document.  (Of course, automated POP would be better, but that's a different story)

---
In reply to: [154235122](https://github.com/Azure/azure-iot-sdk-node/pull/166#discussion_r154235122) [](ancestors = 154235122,154234983)